README.md

```diff
@@ -36,18 +36,20 @@ of ORACC, enabling more projects to adopt it.
 Nammu is currently being developed by the
 [UCL Research Software Development Group](https://www.ucl.ac.uk/research-it-services/about/research-software-development).
 
-<img src="./doc/mockups/nammu_0.5.png" align="center" width="70%">
+<img src="./doc/mockups/nammu_0.6.png" align="center" width="90%">
 
 
-## Validation of ATF files
+## Validation and lemmatisation of ATF files
 
 In order to validate ATF files, Nammu uses two approaches:
-* Online validation against ORACC server.
+* Online validation against ORACC server, configurable on the settings menu.
 * Offline validation using [pyORACC](https://github.com/oracc/pyoracc), the new
 ATF parser developed by UCL RSDT.
 
+Lemmatisation can only be done online with the ORACC server.
 
-#### Validation against the ORACC server
+
+#### Validation and lemmatisation against the ORACC server
 The ORACC server is hosted at University of Pennsylvania (Philadelphia) and
 maintained by [Prof. Steve Tinney](https://www.ling.upenn.edu/people/tinney).
 The ORACC server provides with SOAP web services for validation and lemmatisation
```

Comment:
There's another extra 'with' in the following paragraph too - spotted as I was reading the rendered readme!